src/components/index.ts

```diff
@@ -57,3 +57,4 @@ export { Row, RowProps, Column, ColumnProps } from './Grid/Grid';
 export { Popover, PopoverProps } from './Popover/Popover';
 export { InfoBanner, InfoBannerProps, InfoBannerVariants } from './InfoBanner/InfoBanner';
 export { InfoBannerCard } from './InfoBanner/blocks/InfoBannerCard';
+export { ColorMode } from './ColorMode/ColorMode';
```

Comment:
Shouldn't we export `InvertedColorMode` as well?

```suggestion
export { ColorMode } from './ColorMode/ColorMode';
export { InvertedColorMode } from './ColorMode/InvertedColorMode';
```

Reply:
Nice catch. This is the part I need to finalize